libs/brig-types/src/Brig/Types/Test/Arbitrary.hs

```diff
@@ -313,13 +313,17 @@ instance Arbitrary UserProfile where
         <*> arbitrary
 
 instance Arbitrary RichField where
-    arbitrary = RichField <$> arbitrary <*> arbitrary
+    arbitrary = RichField <$> arbitrary <*> (arbitrary `suchThat` (/= ""))
```

Comment:
mention `normalizeRichInfo` here, and where it is used, so we know what goes wrong if this restriction is missing here.